mne/tests/test_source_estimate.py

```diff
@@ -1668,6 +1668,9 @@ def test_vol_mask():
 @testing.requires_testing_data
 def test_stc_near_sensors(tmp_path):
     """Test stc_near_sensors."""
+    import pdb
+
+    pdb.set_trace()
```

Comment:
Remove cruft

Reply:
Thanks for the review, sorry for the bad mistake!